test/WebSites/ActionResultsWebSite/Controllers/ActionResultsVerificationController.cs

```diff
@@ -108,5 +119,20 @@ private DummyClass CreateDummy()
                 SampleString = "Foo"
             };
         }
+
+        private DisposableDummy CreateDisposableDummy()
+        {
+            return new DisposableDummy();
+        }
+
+        private class DisposableDummy : IDisposable
+        {
+            public static int disposeCallCount = 0;
```

Comment:
Static in a unit test :frowning: :panda_face: Is there a better way of doing this? Like using some other kind of state that isn't for the whole lifetime of the *process*?